package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A Matrix <--> Slack bridge",
   "main": "app.js",
   "scripts": {
-    "postinstall": "npm run build",
+    "prepare": "npm run build",
```

Comment:
This will also run on `npm install`. In my opinion `prepack` would be a better script key to build the code.
https://docs.npmjs.com/misc/scripts

Reply:
I disagree. This will run on npm install, so when people install the package by cloning to their system. This saves an extra instruction of running `npm run build`